cmd/terramate/cli/cli.go

```diff
@@ -52,11 +55,18 @@ const (
 	defaultBranchBaseRef = "HEAD^"
 )
 
+const (
+	defaultLogLevel = "none"
+	defaultLogFmt   = "json"
+)
+
 type cliSpec struct {
 	Version       struct{} `cmd:"" help:"Terramate version."`
 	Chdir         string   `short:"C" optional:"true" help:"sets working directory."`
 	GitChangeBase string   `short:"B" optional:"true" help:"git base ref for computing changes."`
 	Changed       bool     `short:"c" optional:"true" help:"filter by changed infrastructure"`
+	LogLevel      string   `optional:"true" help:"sets the log level. Possible values are: 'trace', 'debug', 'none'. The default is 'none'."`
+	LogFmt        string   `optional:"true" help:"sets the log format. Possible values are: 'json', 'text'. The default is 'text'."`
```

Comment:
here it says the default is "text" but "defaultLogFmt" is "json" (can be seen [here](https://github.com/mineiros-io/terramate/pull/117/files#diff-3a3a4d5e49230d35db58ed019f072d3b3fb7d818e9c255c2208d68bc4bc08578R60) ).

which one is the intended default ?

Reply:
text is intended default

Reply:
ah then this is wrong on this PR (I got JSON as default here). we can fix on a follow up PR (doesn't break anything, just get some JSON =P).